account_invoice_refund_link/views/account_invoice_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2004-2011 Pexego Sistemas Informáticos. (http://pexego.es)
+     Copyright 2014 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+     Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_customer_account_invoice_add_refunds_details_form" model="ir.ui.view">
+    <field name="name">Account invoice (customer) | add Refunds details (form)</field>
+    <field name="model">account.move</field>
+    <field name="inherit_id" ref="account.view_move_form"/>
+    <field name="arch" type="xml">
+        <notebook position="inside">
+            <page name="refunds"
+                  string="Refunds"
+                  attrs="{'invisible':[('type', 'not in', ['in_invoice', 'out_invoice'])]}"
+            >
+                <field name="refund_invoice_ids" nolabel="1">
+                    <tree decoration-info="state == 'draft'"
```

Comment:
Isn't here a standard invoice view to reference instead of hard-coding one?

Reply:
Maybe the intention was to show the reason for the refund, which doesn't appear in the standard Invoice Tree view